WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -364,11 +369,16 @@ private void setComment(int localBlogId, final Comment comment) {
         // notification about a reply to a comment this user posted on someone else's blog
         mIsUsersBlog = (comment != null && WordPress.wpDB.isLocalBlogIdInDatabase(mLocalBlogId));
 
-        if (mIsUsersBlog)
+        if (mIsUsersBlog) {
             mRemoteBlogId = WordPress.wpDB.getRemoteBlogIdForLocalTableBlogId(mLocalBlogId);
+        }
 
-        if (isAdded())
+        if (isAdded()) {
             showComment();
+        }
+
+        // Reset the reply unique id since mComment just changed.
+        setReplyUniqueId();
```

Comment:
And fixed the same kind of String.format/locale problem in `ReaderCommentListActivity` - 85ab7ed